packages/core/src/Utils/Validator/declarative-validation-rules.js

```diff
@@ -181,3 +181,23 @@ export const getPreBuildDVRs = () => {
 };
 
 export const getPasswordLengthConfig = type => ({ min: /^mt$/.test(type) ? 8 : 6, max: 25 });
+
+export const validateMainPassword = password => {
+    const errors = {};
+
+    if (
+        !validLength(password, {
+            min: 6,
+            max: 25,
+        })
+    ) {
+        errors.password = localize('You should enter {{min_number}}-{{max_number}} characters.', {
+            min_number: 6,
```

Comment:
why is this being changed to 6? the new min value is 8

Reply:
> why is this being changed to 6? the new min value is 8

Done